ajax/libs/vue-select/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "vue-select",
+  "filename": "vue-select.js",
+  "version": "1.3.3",
+  "description": "A native Vue.js component that provides similar functionality to Select2 without the overhead of jQuery.",
+  "author": "Jeff Sagal <sagalbot@gmail.com>",
+  "homepage": "https://sagalbot.github.io/vue-select/",
+  "license": "MIT",
+  "keywords": [
+    "vue.js",
+    "vue-select",
+    "select2",
+    "tagging"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/sagalbot/vue-select.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/sagalbot/vue-select.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

Comment:
The point is wrong here, all the files before v1.1.0 shouldn't be added, to prevent adding the files before v1.1.0, or you can say to prevent adding versions before v1.1.0, you should update [.gitignore](https://github.com/cdnjs/cdnjs/blob/master/.gitignore) to ignore them, not to make the auto-update use a specific rule for them, let's fix it.